server/models/questions/addReader.js

```diff
@@ -0,0 +1,39 @@
+const { pool } = require('../database')
+
+/**
+ *
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {String} param0.reader
+ * @param {Number} param0.questionId
+ * @return {Promise}
+ */
+
+function addReader({ username, reader, questionId }) {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      `INSERT INTO questions_editors(question_id, editor, access) 
+       SELECT question_id, ?, ? FROM questions_editors WHERE editor=? AND question_id=?`,
```

Comment:
#### NEW

1. The person **`a`** adding reader **`b`** need not have `write` access... he can do so even with `read` access...so no need of checking access of **`a`**